docs/api-usage.md

```diff
@@ -82,6 +106,9 @@ curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/crede
 }'
 ```
 
+If you are sending a SMS or Telegram campaign, you will need to provide an additional `label` parameter in the request body. This label refers
```

Comment:
Do you mind creating a separate section for SMS and Telegram?

SMS
```
curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/new-credentials/v2' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer your_api_key' 
--data-raw '{
	"twilio_account_sid": "ACxxx",
	"twilio_api_key": "SKxxx",
	"twilio_api_secret": "secret",
	"twilio_messaging_service_sid": "MGxxx",
	"recipient":"+6588888888"
}
'

curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/credentials' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer your_api_key' \
--data-raw '{
	"label":"SavedCredentials",
	"recipient":"+6588888888"
}
'
```

Telegram
```
curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/new-credentials/v2' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer your_api_key' \
--data-raw '{
   "telegram_bot_token":"123123123123:xxxxxx"
}
'

curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/credentials' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer your_api_key' \
--data-raw '{
    "label": "SavedBot"
}
'
```

Reply:
Ok! Will update the docs.